installer/efs-provisioner/templates/deployment.yaml

```diff
@@ -0,0 +1,122 @@
+# Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
Any reason not to use https://github.com/helm/charts/tree/master/stable/efs-provisioner? I find it too much to add the helm charts here. We should keep it as an external dependency as we do with Spark, Flink etc.

Reply:
I understand your concern. However if I remember correctly, I had to tweak a little bit the helm chart that you are referring too. For instance, the `ServiceAccount` it's already created by the original install scripts. Not sure if it is possible to import only certain components of a given helm char. If you know how to do that I am happy to take your advice and put that change in place :)

Reply:
Let's go one by one the issues you faced when tried the external helm charts.
For example regarding the service account you can pass a name [here](https://github.com/helm/charts/blob/master/stable/efs-provisioner/values.yaml]) and you can also select if a service account should be created by setting this [flag](https://github.com/helm/charts/blob/4c39ce95e175cb1d75fab43da0c799186b4c0387/stable/efs-provisioner/values.yaml#L62) to `false`.

Reply:
@skonto I think you are right, maybe when I started the `efs-provisioner` i was  _hacking_ the chart rather than understand it. Let me give it a second pass and see how far I get.